app/test_main.py

```diff
@@ -1 +1,77 @@
-# write your code here
+from typing import Callable
+
+import pytest
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@pytest.fixture()
+def mocked_valid_url() -> None:
+    with mock.patch("app.main.valid_google_url") as mock_valid_url:
+        yield mock_valid_url
+
```

Comment:
The name of the fixture `mocked_valid_url` is not descriptive enough. It would be better to rename it to `mock_valid_google_url` to match the function it's mocking.